views/random-beers.hbs

```diff
@@ -0,0 +1,25 @@
+
+
+
+<section>
+<div>
+    {{> randombeerpartial }}
+   <p><h1>{{food_pairing}}</h1> </p>
```

Comment:
You're not displaying the food_pairing and brewer_tips here.
For it to work, you should've used the each loop here.

```
<section>
    <div>
    {{> randombeerpartial }}
        {{#each randomBeers}}
    <p><h1>{{food_pairing}}</h1> </p>
    <p><a>{{brewers_tips}}</a></p>
        {{/each}}
    </div>
</section>
```